bindings/python/install_data/libtorrent/__init__.pyi

```diff
@@ -0,0 +1,1954 @@
+import datetime
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Iterable
+from typing import Iterator
+from typing import List
+from typing import Mapping
+from typing import Optional
+from typing import overload
+from typing import Tuple
+from typing import Union
+
+from typing_extensions import TypedDict
+
+# I created this file by starting with stubgen and modifying its output.
+
+# I recommend that maintainers occasionally re-run stubgen, and compare it to
+# this file. This diff will be too verbose to use for small changes, but it's the
+# only way to ensure coverage.
+
+# To help with the above process, please keep everything in sorted order, to
+# match stubgen's output.
+
+# Typing notes:
+# - This project has a very particular approach to typed enum values. Pay
+#   special attention to which are typed, and which are just ints.
+# - Some attributes like add_torrent_params.save_path ought to be read as one
+#   type (str) and set as another (Union[str, bytes]), but mypy doesn't support
+#   this.
```

Comment:
Seems like a fine thing to support, but I believe `add_torrent_params.save_path` doesn't actually support being set from a `pathlib.Path` instance today, so it would require a code change.

The type stubs should (mostly) faithfully reflect the implemented functionality.

In this comment, I was referring to this situation:
```python
atp.save_path = "str"  # supported, should type-check
atp.save_path = b"bytes"  # supported, should type-check
x = atp.save_path  # type-checker should know this is a *str*
```

But, mypy just doesn't support this today. Best we can do is make `save_path` a `str` since that's the most common usage.